tracing/src/main/java/io/strimzi/test/tracing/TracingUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.tracing;
+
+import org.apache.kafka.streams.KafkaClientSupplier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Tracing initialization
+ */
+public class TracingUtil {
+    private static final Logger log = LogManager.getLogger(TracingUtil.class);
```

Comment:
We would normally use uppercase for final static constants. You seem to use it for `HANDLES` as well.

@im-konge This is ideally something what checkstyle should check.

Reply:
Sure, will add it there. Good catch